HelloWorld/reunioncppdesktopsampleapp/ReunionCppDesktopSampleApp/ReunionCppDesktopSampleApp (Package)/ReunionCppDesktopSampleApp (Package).wapproj

```diff
@@ -1,71 +1,71 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
>< [](start = 0, length = 1)

Anyone know why git has such a messed up view of how this file was changed? I believe all that was changed <version> in this line:
`<PackageReference Include="Microsoft.ProjectReunion" Version="<version>">`